metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/Main.scala

```diff
@@ -0,0 +1,78 @@
+package it.pagopa.interop.metricsreportgenerator
+
+import cats.implicits._
+import com.typesafe.scalalogging.Logger
+import io.circe.syntax._
+import it.pagopa.interop.catalogmanagement.model.CatalogItem
+import it.pagopa.interop.commons.files.service.FileManager
+import it.pagopa.interop.commons.utils.TypeConversions.EitherOps
+import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
+import it.pagopa.interop.metricsreportgenerator.models.Metric
+import it.pagopa.interop.metricsreportgenerator.repository.impl.{CatalogRepositoryImpl, TenantRepositoryImpl}
+import it.pagopa.interop.metricsreportgenerator.repository.{CatalogRepository, TenantRepository}
+import it.pagopa.interop.metricsreportgenerator.util.Utils.{createMetric, getMeasurableEServices}
+import it.pagopa.interop.metricsreportgenerator.util.{ApplicationConfiguration, FileUtils}
+import org.mongodb.scala.MongoClient.DEFAULT_CODEC_REGISTRY
+import org.mongodb.scala.connection.NettyStreamFactoryFactory
+import org.mongodb.scala.{ConnectionString, MongoClient, MongoClientSettings}
+
+import java.util.concurrent.{ExecutorService, Executors}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor, Future}
+object Main extends App {
+
+  private val logger: Logger = Logger(this.getClass)
+
+  logger.info("Starting metrics report generator job")
+
+  val blockingThreadPool: ExecutorService = Executors.newFixedThreadPool(Runtime.getRuntime.availableProcessors())
+  implicit val blockingEC: ExecutionContextExecutor = ExecutionContext.fromExecutor(blockingThreadPool)
+
+  implicit val client: MongoClient = MongoClient(
```

Comment:
in `cqrs` model of commons there is a helper to access read model that out of the box configures the client, select the collection and decodes the results. If possible I suggest to use that

Reply:
My fault I didn't remember about that module...
Thank you!